### DISCUSSION TITLE
📆 2023-11-02 周四-文字会议

### DISCUSSION THREAD

**Discussion by talkbear** [RemoteGroupLabs/frontend-code-camp#2] [General]
## 🎉 会议流程
> 🍀 文字会议采取小组讨论方式，一是方便文字记录，二是熟悉掌握GitHub相关模块使用。

1. 参与人**第一条发布**须为【进度分享】（具体格式参考 👇 ）。
2. **主持人**或其他**成员**直接在他人的【进度分享】下回复，参与讨论。
3. 完成当天会议任务

- [x] 下载当前库到本地
- [x] 新建独立分支并推送
- [x] 加入Github小组（邮箱通过） 
- [x] 访问 ☝️ 上面Discussioons, Projects, Wiki模块。

5. **主持人**发布【会议总结】，会议结束。

## 【进度分享】格式

```
本周任务：完成GitHub小组初始化

**已完成**
- 代码库新建
- Projects新建
- Wiki新建
- 小组讨论新建

**未完成**
- 学习计划跟踪

**正在攻关问题**
- Wiki文档完善

```

**Comment by zed11213** [comment]
本周任务：盲盒接口调用API开发

**已完成**
- 代码库新建
- Projects新建

**未完成**
- API调用

**正在攻关问题**
- 学习TS文件

**Reply by talkbear** [reply]
正在攻关问题也可以写上，你遇到的不好解决的问题。

**Comment by ChiYmiya** [comment]
本周任务：自主学习

**已完成**
学习进度勾选

**未完成**
学习路径细节补充了解
tailwindcss初体验

**正在攻关问题**
自主学习

**Reply by talkbear** [reply]
记得，完成几个小任务:-

**Comment by LDW89** [comment]
本周任务：完成html和css进度

**已完成**
- html
- css基础


**未完成**
- css盒模型和响应式布局

**正在攻关问题**
- css盒模型和响应式布局

**Reply by talkbear** [reply]
作业也做了吗，如果提交分支了，可以吧commit贴出来，大家相互学习下。

**Reply by talkbear** [reply]
另外，记得把上面几个小任务做了吧，方便后面提交作业和审批。

**Comment by piprot** [comment]
本周任务：HTML Basics+HTML5和语义化标签+html Tables

**已完成**
- [Introduction_to_HTML](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML)
- [Multimedia_and_embedding](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding)
- [HTML速成（可选）](https://www.bilibili.com/video/BV1vs411M7aT/?spm_id_from=333.337.search-card.all.click)
- [HTML5](https://zh.wikipedia.org/wiki/HTML5)
- [HTML之语义化标签(HTML5 Markup)](https://juejin.cn/post/7141305668891312164)
- [Tables](https://developer.mozilla.org/zh-CN/docs/Learn/HTML/Tables)

**未完成**
- [作业-完成谷歌html页面（线上完成）](https://jsfiddle.net/jasmineeven2/Lrq9o1bf/14/)
- [作业-完成谷歌html页面（github，本地完成)](https://github.com/talkbear/go-japan-fe-practices/blob/main/%E5%88%9B%E5%BB%BA%E4%B8%80%E4%B8%AA%E5%AE%8C%E6%95%B4%E7%9A%84html%E6%96%87%E4%BB%B6%20%E5%8F%82%E8%80%83%E7%BD%91%E7%AB%99Google%E5%AE%98%E7%BD%91.html)
- [作业-编写语义化版本的Google](https://jsfiddle.net/talkbear/m5btzLrd/3)
- [作业-编写表格](https://jsfiddle.net/jasmineeven2/Lrq9o1bf/13/)

**正在攻关问题**
- [作业-完成谷歌html页面（线上完成）](https://jsfiddle.net/jasmineeven2/Lrq9o1bf/14/)

**Comment by facioplegia** [comment]
本周任务：HTML基础学习

<h3>已完成</h3>
<ol>
<li>HTML基础了解</li>
<li>部分语义化标签认知</li>
<li>Wiki新建</li>
<li></li>
</ol>
<br></br>
<h3>未完成</h3>
<ul>
<li>Google页面代码编写</li>
</ul>
<h3>正在攻关问题</h3>
<ol>
<li>git本地库拉取&新建分支</li>
<ol>

**Comment by talkbear** [comment]
## 【会议总结】

主要需要大家熟悉流程，并完成几个小任务。目前任务完成情况：
```
 - [ ] 下载当前库到本地
 - [ ] 新建独立分支并推送
 - [ ] 加入Github小组（邮箱通过）
 - [ ] 访问 ☝️ 上面Discussioons, Projects, Wiki模块。
```
- [x] 张洧铭
- [x] 杜淳滢
- [x] 汪宏伟
- [x] 祁静波
- [x] 李道伟

后面大家完成作业后，尽快提交代码到自己的分支上。
